src/plugins/banger.tsx

```diff
@@ -16,8 +16,16 @@ export default definePlugin({
             find: "BAN_CONFIRM_TITLE.",
             replacement: {
                 match: /src:\w\(\d+\)/g,
-                replace: 'src: "https://i.imgur.com/wp5q52C.mp4"'
+                replace: "src: Settings.plugins.BANger.source"
```

Comment:
shouldn't this be 
```suggestion
                replace: "src: Vencord.Settings.plugins.BANger.source"
```
remember that not everyone has console shortcuts enabled

Reply:
Ah I did not know that was apart of console shortcuts

Reply:
for future ref the only guaranteed window props are Vencord and VencordNative, anything else is console shortcuts